examples/deps/repositories.bzl

```diff
@@ -14,6 +24,70 @@ def repositories():
         type = "zip",
     )
 
+    http_archive(
+       name = "cmake",
+       build_file_content = cmake_all_content,
+       strip_prefix = "cmake-3.19.4",
+       sha256 = "7d0232b9f1c57e8de81f38071ef8203e6820fe7eec8ae46a1df125d88dbcc2e1",
+       urls = ["https://github.com/Kitware/CMake/releases/download/v3.19.4/cmake-3.19.4.tar.gz"],
+    )
+
+    http_archive(
+        name = "cmake_linux_aarch64",
+        sha256 = "c6c6e5d7a1b8745a6780e04804d5cbae5cc1bb04c89a3b8a6c1d8ccf24f5cdaf",
+        urls = [
+            "https://cmake.org/files/v3.19/cmake-3.19.4-Linux-aarch64.tar.gz",
+        ],
+        strip_prefix = "cmake-3.19.4-Linux-aarch64",
+        build_file_content = all_content,
+    )
+
+    http_archive(
+        name = "cmake_linux_x86_64",
+        sha256 = "ff23e1f53c53e8ef1fa87568345031d86c504e53efb52fa487db0b8e0ee4d3ff",
+        urls = [
+            "https://cmake.org/files/v3.19/cmake-3.19.4-Linux-x86_64.tar.gz",
+        ],
+        strip_prefix = "cmake-3.19.4-Linux-x86_64",
+        build_file_content = all_content,
+    )
+
+    http_archive(
+        name = "cmake_macos_universal",
+        sha256 = "eb1f52996632c1e71a1051c9e2c30cc8df869fb5a213b1a0d3b202744c6c5758",
+        urls = [
+            "https://cmake.org/files/v3.19/cmake-3.19.4-macos-universal.tar.gz",
+        ],
+        strip_prefix = "cmake-3.19.4-macos-universal",
```

Comment:
Nit: Would it not be better to also strip `CMake.app/Contents`?

Reply:
By having the `CMake.app` in the folder structure this causes macOS to treat this as a application bundle. I don't know if removing this from the folder name breaks anything?

Reply:
Bazel is never going to execute it as an application bundle. I know from experience that you can remove this.